mapzen-places-api/src/main/java/com/mapzen/places/api/internal/PlaceDetailFetcher.java

```diff
@@ -1,29 +1,32 @@
 package com.mapzen.places.api.internal;
 
-import com.mapzen.tangram.LngLat;
-
 import java.util.Map;
 
 /**
  * Interface for fetching details about a place.
  */
 interface PlaceDetailFetcher {
 
+  String PROP_ID = "id";
+  String PROP_AREA = "area";
+
+  String PELIAS_GID_BASE = "openstreetmap:venue:";
```

Comment:
just confirming if we know that the pelias gid for the place end point always is a `venue`?